contracts/sg721-base/src/contract.rs

```diff
@@ -67,13 +67,16 @@ where
             description: msg.collection_info.description,
             image: msg.collection_info.image,
             external_link: msg.collection_info.external_link,
+            start_trading_time: None,
```

Comment:
can't be set during instantiation? doesn't seem to be taking the information from the `InstantiateMsg`